.github/workflows/check-bootstrap-testnet-full-sync.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Check Bootstrap (testnet,full-sync)'
+
+on:
+  # TODO(marun) Add a schedule
+  workflow_dispatch:
+
+jobs:
+  check_bootstrap_testnet_full_sync:
+    name: Check Bootstrap (testnet,full-sync)
+    runs-on: avalanche-avalanchego
+    timeout-minutes: 7200 # 5 days, maximum allowed for jobs running on a self-hosted runner
```

Comment:
q: I guess testnet is much faster to sync than mainnet. Should we try a smaller timeout here, just to see this job failing earlier than mainnet one if anything weird happens?